website/react-magma-docs/src/pages/api/grid.mdx

```diff
@@ -0,0 +1,196 @@
+---
+pageTitle: Grid API
+title: Grid
+props:
+  - GridProps
+  - Grid
```

Comment:
```
props: 
  - GridProps
  - GridItemProps
```

this is just the magic that maps the typescript props to a component for this page we don't need `Grid` or `GridItem` in it